src/common-asserts/assert-requestCount.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="../../../general/schematron/NictizTestScript.sch" type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<!-- additionalDescription: Additional description to the assert. -->
+<!-- warningOnly: Set the assert to warningOnly -->
+<!-- use only for use case 'send' in PHR scripts -->
+<nts:component  xmlns="http://hl7.org/fhir" xmlns:nts="http://nictiz.nl/xsl/testscript">
+    <nts:parameter name="additionalDescription" value=""/>
+    <nts:parameter name="warningOnly" value="false"/>
+    <action>
+        <assert>
+            <description value="Confirm that the returned searchset Bundle contains {$count} {$resource} resource(s). {$additionalDescription}"/>
```

Comment:
Dit is nog niet aangepast. De description gaat hier nog over de returned searchset Bundle.